spec/schema.rb

```diff
@@ -40,3 +52,11 @@ class UUIDTenant < ActiveRecord::Base
 class UUIDEmployee < ActiveRecord::Base
   belongs_to :tenant, class_name: 'UUIDTenant', foreign_key: :tenant_id
 end
+
+class Company < ActiveRecord::Base
+  has_many :employees, class_name: 'CompanyEmployee', foreign_key: :company_id
+end
+
+class CompanyEmployee < ActiveRecord::Base
+  belongs_to :company
+end
```

Comment:
The policy you create need to be tested to ensure they work correctly. It is a good practice to add a single simple test case to `spec/activerecord-tenant-level-security/tenant_level_security_spec.rb`.

Reply:
My response was missed. I added it in this commit https://github.com/kufu/activerecord-tenant-level-security/pull/22/commits/5f7056473808597679a7eff273307180014297b7